test/luatest_helpers/utils.lua

```diff
@@ -0,0 +1,11 @@
+local fio = require('fio')
+
+local utils = {
+    SOCKET_DIR = fio.abspath(os.getenv('VARDIR') or 'test/var')
+}
+
+function utils.get_instance_uri(alias)
+    return ('%s/%s.iproto'):format(utils.SOCKET_DIR, alias)
+end
```

Comment:
Why it is not a server object method?

Reply:
Also I would note that we generally avoid `get_` prefix for accessors.

Reply:
Fixed, moved to server.lua.